risc0/zkvm/src/host/client/posix_io.rs

```diff
@@ -34,7 +34,7 @@ impl<'a> Default for PosixIo<'a> {
             read_fds: Default::default(),
             write_fds: Default::default(),
         };
-        new.with_read_fd(fileno::STDIN, BufReader::new(stdin()))
+        new.with_read_fd(fileno::STDIN, Cursor::new(vec![]))
             .with_write_fd(fileno::STDOUT, stdout())
             .with_write_fd(fileno::STDERR, stderr());
```

Comment:
So it seems like we should probably not use default stdout and stderr as well if we're going to stop using stdin.

Reply:
So I thought about this, and my inclination is the indeed have the default be empty vec for stdin and pipe to stdout. It is inconsistent, but I think it's what a new user will find least surprising / most useful. In particular, this preserves the behavior that `println!` and other guest output goes to users terminal. Do you agree with that logic?